includes/payment-methods/class-wc-stripe-upe-payment-method.php

```diff
@@ -30,7 +30,7 @@ abstract class WC_Stripe_UPE_Payment_Method {
 	 *
 	 * @var string
 	 */
-	protected $title;
```

Comment:
Note that when I changed the `WC_Stripe_UPE_Payment_Method` class to extend `WC_Payment_Gateway` I had to make all these properties public to be consistant with that class.